### PR TITLE
linux-eic-shell.yml: run trigger-container for when merge_group is tested

### PR DESCRIPTION
**Pull request by veprbl**
Since we now require container build to pass as a requirement for successful merge to main, we need to actually attempt to build it.

### CODE REVIEW

**Comment by veprbl** [comment]
@simonge Can you approve this? This is safe (tm).

**Review by simonge — DISMISSED** [review]
Trusting...

**Comment by veprbl** [comment]
I've changed timeout from 3h to 6h. Let's try again.

**Comment by veprbl** [comment]
Not going to work https://github.com/eic/EICrecon/actions/runs/8819319775/job/24210808245
What we use in eic/epic does not require a synced branch

**Comment by veprbl** [comment]
This needs
https://github.com/eic/detector_benchmarks/blob/16fd651e6f9ce0c670eefe047d337cbe7bbc8296/.github/workflows/mirror.yaml#L19-L29
plus, if I remember correctly, also merge group ref somehow didn't work, so a fix is needed.

I will disable the container requirement for now.

File: .github/workflows/linux-eic-shell.yml [hunk 1]
```diff
@@ -884,7 +884,7 @@ jobs:
 
   trigger-container:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name != 'merge_group' && github.actor != 'dependabot[bot]' }}
+    if: ${{ github.actor != 'dependabot[bot]' }}
```
Comment (wdconinc) [inline .github/workflows/linux-eic-shell.yml:887]: While we're here:
```suggestion
    env: 
      secret: ${{ secrets.EICWEB_CONTAINER_TRIGGER }}
    if: ${{ github.actor != 'dependabot[bot]' && secret != '' }}
```

Comment (veprbl) [reply .github/workflows/linux-eic-shell.yml:887]: Why can't we compare the value directly?

Comment (veprbl) [reply .github/workflows/linux-eic-shell.yml]: Okay, I was confused, this is not a `step.*.if`, but a `jobs.*.if`, where `env` should not be accessible, allegedly.

**Comment by veprbl** [comment]
@wdconinc Let's try again, since, I think, I resolved the 404 issue. Please, approve.

**Review by wdconinc — APPROVED** [review]
Another one for the history books!

**Comment by veprbl** [comment]
We require eicweb benchmarks for eic/epic repository merges, yet somehow here it failed. In addition to container build failures and slowness, I don't think this is reliable enough to enable this now.

**Pull request closed without merging**